test/suites/standard/log-scan.robot

```diff
@@ -0,0 +1,51 @@
+*** Settings ***
+Documentation       Tests for various log messages we do or do not want.
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+Library             ../../resources/journalctl.py
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+Test Tags           restart    slow
+
+
+*** Variables ***
+${CURSOR}       ${EMPTY}    # The journal cursor before restarting MicroShift
+
+
+*** Test Cases ***
```

Comment:
I'd add `Timed out waiting for services to stop` ([src](https://github.com/openshift/microshift/blob/69ca1dd7625190814b97d70091c6d9b62be391a6/pkg/cmd/run.go#L265)) and perhaps `systemd[1]: microshift.service: start operation timed out. Terminating.` to catch when we start exceeding deadlines. Though the latter is more suitable for Pablo's idea.
We should be more strict in tests than in prod (e.g.: checking for pod restarts via greenboot in tests is fine to catch problems, but maybe overkill in prod)

Reply:
I can do the first one, since we emit that. The other is a systemd message and I'll have to do some research about how to collect that since it won't be in the MicroShift logs. Can we do that in a separate ticket?

Reply:
Of course